src/ts/content.ts

```diff
@@ -1,12 +1,20 @@
-import * as fileIcons from 'file-icons-js';
 import * as domLoaded from 'dom-loaded';
 import select from 'select-dom';
 import mobile from 'is-mobile';
 import { observe } from 'selector-observer';
+import {
+  getClass,
+  getClassWithColor,
+  getClassWithDarkColor,
+} from '../file-icons/file-icons';
 
 import { StorageKey } from './background';
 import '../css/icons.css';
 
+import '../file-icons/css/fonts.css';
```

Comment:
Just for reducing the lines of this file, can we move these three lines into `css/icons.css` as before?